funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -397,9 +376,19 @@ def register_endpoint(funcx_client, endpoint_name, endpoint_uuid, endpoint_dir):
     logger.debug("Attempting registration")
     logger.debug(f"Trying with eid : {endpoint_uuid}")
     import funcx_endpoint
-    reg_info = funcx_client.register_endpoint(endpoint_name,
-                                              endpoint_uuid,
-                                              endpoint_version=funcx_endpoint.__version__)
+    if previously_registered:
```

Comment:
Let's skip checking for previously_registered until we have better info from register_endpoint as to why it failed.

Reply:
The `previously_registered` logic has been removed.